scripts/init-submodules-no-riscv-tools-nolog.sh

```diff
@@ -81,3 +81,5 @@ echo "# line auto-generated by init-submodules-no-riscv-tools.sh" >> env.sh
 echo '__DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]:-${(%):-%x}}")")"' >> env.sh
 echo "PATH=\$__DIR/bin:\$PATH" >> env.sh
 echo "PATH=\$__DIR/software/firemarshal:\$PATH" >> env.sh
+echo "-Dsbt.sourcemode=true" > .sbtopts
```

Comment:
Can you delete the `.sbtopts` file in the main repo area?

Reply:
Once that is done, I'll open this PR in our CI to double-check that things work.